From be115301a1f6ac991fb883434813e9f6ab3a88ca Mon Sep 17 00:00:00 2001
From: Cosmin-Gabriel Samoila <cosmin.samoila@nxp.com>
Date: Wed, 14 Nov 2018 15:01:23 +0200
Subject: [PATCH 5125/5242] MLKU-24-3 sound:asoc: fix missing space after if
 checkpatch errors

commit  ecd01c473d3a54987b3ea554088eab42901af6c9 from
https://source.codeaurora.org/external/imx/linux-imx.git

In order to upstream the micfil driver, we need to fix
missing space after if errors and warnings reported
by checkpatch.

Signed-off-by: Cosmin-Gabriel Samoila <cosmin.samoila@nxp.com>
Reviewed-by: Daniel Baluta <daniel.baluta@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_micfil.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/fsl/fsl_micfil.c b/sound/soc/fsl/fsl_micfil.c
index 6f1dbe8..4f2e9aa 100644
--- a/sound/soc/fsl/fsl_micfil.c
+++ b/sound/soc/fsl/fsl_micfil.c
@@ -1377,7 +1377,7 @@ static int fsl_micfil_set_mclk_rate(struct fsl_micfil *micfil, int clk_id,
 		return -EINVAL;
 	}
 
-	while(p) {
+	while (p) {
 		struct clk *pp = clk_get_parent(p);
 
 		if (clk_in_list(pp, micfil->clk_src)) {
-- 
1.7.9.5

